packages/wallet-core/src/utils/getTransactionByHash.ts

```diff
@@ -0,0 +1,6 @@
+import { Client } from '@chainify/client';
+import { Transaction } from '@chainify/types';
+
+export async function getTransactionByHash(client: Client, hash: string): Promise<Transaction<any>> {
```

Comment:
What's the purpose of this extra level of indirection?
Just makes it a bit harder to figure out what the actual call is

Reply:
Done